flight_phase_files/HALO_RF05_20200128_info.yaml

```diff
@@ -19,59 +19,125 @@ remarks:
 - Rather long excursion towards NTAS with radar calibration maneuver.
 - Sugar cloud patterns within the circle.
 segments:
-- kind: circle
-  name: c1
-  start: 2020-01-28 15:21:54
-  end: 2020-01-28 16:13:03
-- kind: circle_break
-  name: cb1to2
-  start: 2020-01-28 16:13:03
-  end: 2020-01-28 16:32:33
-- kind: circle
-  name: c2
-  start: 2020-01-28 16:32:33
-  end: 2020-01-28 17:25:06
-- kind: circle_break
-  name: cb2to3
-  start: 2020-01-28 17:25:06
-  end: 2020-01-28 17:42:40
-- kind: circle
-  name: c3
-  start: 2020-01-28 17:42:40
-  end: 2020-01-28 18:37:26
-- kind: straight_leg
-  name: leg1_outbound_NTAS
+- kinds:
+  - circle
+  name: Circle 1
+  irregularities: []
+  segment_id: HALO-0128_c1
+  start: 2020-01-28 15:20:54
+  end: 2020-01-28 16:17:27
+  good_dropsondes: 12
+- kinds:
+  - circle_break
+  name: Break between circle 1 and circle 2
+  irregularities: []
+  segment_id: HALO-0128_cb1
+  start: 2020-01-28 16:17:27
+  end: 2020-01-28 16:31:33
+- kinds:
+  - circle
+  name: Circle 2
+  irregularities: []
+  segment_id: HALO-0128_c2
+  start: 2020-01-28 16:31:33
+  end: 2020-01-28 17:28:35
+  good_dropsondes: 12
+- kinds:
+  - circle_break
+  name: Break between circle 2 and circle 3
+  irregularities: []
+  segment_id: HALO-0128_cb2
+  start: 2020-01-28 17:28:35
+  end: 2020-01-28 17:38:40
+- kinds:
+  - circle
+  name: Circle 3
+  irregularities:
+  - Circle starts 4 minutes (instead of regularly 1 minute) before the first dropsonde.
+  segment_id: HALO-0128_c3
+  start: 2020-01-28 17:38:40
+  end: 2020-01-28 18:37:38
+  good_dropsondes: 11
+- kinds:
+  - circling
+  name: Circling 1
+  irregularities: []
+  segment_id: HALO-0128_o1
+  start: 2020-01-28 15:20:54
+  end: 2020-01-28 18:37:38
+  good_dropsondes: 35
+- kinds:
+  - straight_leg
+  name: Outbound leg towards NTAS
+  irregularities: []
+  segment_id: HALO-0128_sl1
   start: 2020-01-28 18:41:47
   end: 2020-01-28 19:01:46
-- kind: radar_calibration_tilted
-  name: radar_cal_tilted
+- kinds:
+  - radar_calibration_tilted
+  name: Radar calibration
+  irregularities: []
+  segment_id: HALO-0128_rct1
   start: 2020-01-28 19:07:44
   end: 2020-01-28 19:25:26
-- kind: straight_leg
-  name: leg2_inbound_NTAS
+- kinds:
+  - straight_leg
+  name: Inbound leg from NTAS
+  irregularities: []
+  segment_id: HALO-0128_sl2
   start: 2020-01-28 19:29:47
```

Comment:
Same as for HALO-0128_sl1: segment could start a couple of minutes earlier, but maybe this is too much...